test/integration/test_visual_recognition_v4.py

```diff
@@ -116,7 +117,7 @@ def test_04_objects_and_training(self):
         # assert add_images_result is not None
         # image_id = add_images_result.get('images')[0].get('image_id')
         # assert image_id is not None
-        image_id = '1280px-Giraffe_Ithala_KZN_South_202349062c2307571a3f7edc71fe819f'
+        image_id = 'giraffe_00_202349062c2307571a3f7edc71fe819f'
```

Comment:
Any way to add a new image to the collection and add training data to that image and deleting that image instead of changing the training on an image in our current collection?

Reply:
let me update